### PR TITLE
HlsJs Peer5 support

### PR DESCRIPTION
**Pull request by costa**
When the Peer5 plugin is not configured, or configured incorrectly, or fails to initialise, it's all back to fallback.

### CODE REVIEW

**Comment by kaltura-hooks** [comment]
Hi @costa,
Thank you for contributing this pull request!
Please sign the Kaltura CLA so we can review and merge your contribution.
Learn more at http://bit.ly/KalturaContrib

File: modules/Hlsjs/Hlsjs.json [hunk 1]
```diff
@@ -4,7 +4,10 @@
             "resources/hls.min.js" ,
           "resources/hlsjs.js"
         ],
-        "dependencies": "mw.KBasePlugin",
+        "dependencies": [
+            "mw.KBasePlugin",
+            "mw.Peer5"
```
Comment (itaykinnrot) [inline modules/Hlsjs/Hlsjs.json]: @costa  please remove Peer5 from the default hlsjs dependencies and move it to the peer5 module. - we can't merge this code. thanks,

**Comment by costa** [comment]
@itaykinnrot Looks like you were right, it does work this way. Hope there're no more problems with this PR. Thank you!

**Comment by costa** [comment]
@itaykinnrot ping

**Comment by costa** [comment]
@itaykinnrot ping

**Pull request merged**